libunwind/cmake/config-ix.cmake

```diff
@@ -26,71 +26,6 @@ if (NOT LIBUNWIND_USE_COMPILER_RT)
   endif ()
 endif()
 
-# libunwind is using -nostdlib++ at the link step when available,
```

Comment:
Let's split this one into a patch that we can try landing immediately. I suspect we might learn why this code block exists and it would be better not to tie it to the GCC 15 migration.